ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -15,6 +15,24 @@
 # --------------- #
 
 
+# batched_outer
+@st.composite
+def _batched_outer_data(draw):
+    shape = draw(helpers.get_shape(min_num_dims=2, max_num_dims=3))
+    tensors_num = draw(helpers.ints(min_value=1, max_value=5))
+    dtype, tensors = draw(
+        helpers.dtype_and_values(
+            num_arrays=tensors_num,
+            available_dtypes=["float32", "float64"],
```

Comment:
Why can't we use `helpers.get_dtypes("float")` here?

Reply:
because `numpy` backend does not support `bfloat16` so this makes the tests fails.

Reply:
Which numpy function is causing the test to fail? Ideally, `bfloat16` should be in the `unsupported_dtypes` for that function.